install/upgrade/update_to_dev.php

```diff
@@ -180,6 +180,7 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addKey($table, 'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
    // All policies exist for Android
    $migration->addPostQuery("UPDATE `$table` SET `is_android_policy` = '1'");
+   $migration->addPostQuery("DELETE FROM `$table` WHERE `symbol` IN ('disableFmRadio','disableVoiceMail','disableCallAutoAnswer','disableVoiceDictation','disableUsbOnTheGo')");
```

Comment:
We need to remove tasks using these policies too

Reply:
do you mean the cron task, don't you?

Reply:
No, I mean the tables having a foreign key to policies.

All tables having a foreign key to policies miust be cleaned up. If I remember correctly, only glpi_plugin_flyvemdm_tasks has such key

Also, we must send MQTT messages to delete topics of these policies, if any.